src/client/components/switcherSound/switcherSound.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component, Fragment } from 'react';
+import './switcherSound.css';
+import Slider from 'react-rangeslider';
+import { connect } from 'react-redux';
+
+class SwitcherSound extends Component {
+  state = {
+    volumeValueTrack: 0.5,
+    volumeValueVoice: 0.5,
+    track: null,
+    voice: null,
+  };
+
+  static getDerivedStateFromProps(props, state) {
+    const { audioData } = props;
```

Comment:
You could also add here more destructuring 
‘ const { audioData: sound, etc } = props

Reply:
done